content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+# dvc.api.params_show()
+
+Returns all the <abbr>params</abbr> associated with the given <abbr>DVC
+repository</abbr>.
+
+```py
+def params_show(
+    *targets: str,
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
```

Comment:
> It's still actually optional, right? Python syntax may be confusing here.

Optional is used in type hints to indicate that `None` is also a valid value. From https://docs.python.org/3/library/typing.html#typing.Optional:

> Note that this is not the same concept as an optional argument, which is one that has a default. An optional argument with a default does not require the Optional qualifier on its type annotation just because it is optional.

Reply:
Thanks for the reminder!

> an optional argument, which is one that has a default 

But so `*targets` is also optional, right? Even when it has no default value. Nothing is sent in the Usage block below.

Reply:
> But so *targets is also optional, right?

It is optional, but the recommended syntax for the `*args` typing is to include only the valid type (in this case, str). `Optional[str]` would mean that `params_show(None)` would be a valid call, which is not.